TF_Settings_Web/src/Components/Pages/Camera/QuickSetup/Calibration/CalibrationScreen.tsx

```diff
@@ -119,6 +123,25 @@ const CalibrationScreen: React.FC<CalibrationScreenProps> = ({ isHandPresent, on
         };
     }, [displayHandIndicator, progressToClick]);
 
+    const spacebarListener = (event: KeyboardEvent) => {
+        switch (event.code) {
```

Comment:
🙈 If we are only checking one case then an if statement would be more concise here